### PR TITLE
get back Copernicus ECV tool in Galaxy

### PR DESCRIPTION
**Pull request by annefou**
- Open this PR (to make sure I do not forget!).
- Still need to fix creation of .cdsapirc using github secret and pass it to planemo. See
https://github.com/bgruening/galaxytools/blob/master/.github/workflows/pr.yaml#L305
we can give planemo a job_conf
and in the job_conf we can set the ENV

### CODE REVIEW

File: tools/essential_climate_variables/essential_climate_variables.xml [hunk 1]
```diff
@@ -1,25 +1,26 @@
-<tool id="cds_essential_variability" name="Copernicus Essential Climate Variables" version="0.2.0">
+<tool id="cds_essential_variability" name="Copernicus Essential Climate Variables" version="0.2.0" profile="20.05">
```
Comment (bgruening) [inline tools/essential_climate_variables/essential_climate_variables.xml:1]: please bump the tool version as well.

Comment (annefou) [reply tools/essential_climate_variables/essential_climate_variables.xml:1]: I think I already did it a long time ago e.g. 0.2.0 has never been published in the toolshed. Should I still update it?

Comment (bgruening) [reply tools/essential_climate_variables/essential_climate_variables.xml:1]: No, then it is ok. Thanks.

**Comment by annefou** [comment]
@bgruening I can't believe it! Maybe I managed to make it work?!

**Comment by bgruening** [comment]
Cool, yes, looks like it. But now the credentials files will be uploaded to the toolshed I think. We should prevent this by excluding the upload of this file: https://planemo.readthedocs.io/en/latest/standards/docs/best_practices/shed_yml.html?highlight=exclude#shed-upload-includes-excludes

I'm not sure this is all due to a Galaxy bug that does not respect the ENVs anymore :-(

Ready to go from my side if you exclude the credentials file from being uploaded to the toolshed.

**Pull request merged**